recipes/rb-nokogiri/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "nokogiri" %}
+{% set version = "1.10.4" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: fbf91c487d7ceb2b0593384f175843f99152322754f6ff267740bff031cead5a
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem --
+        --use-system-libraries
+    - gem unpack {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
+  skip: {{ win }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ruby
+    - libxml2
+    - libxslt
+    - rb-mini_portile2 >=2.4.0,<2.5
```

Comment:
This might be unnecessary during build-time because we are using libxml2 and libxslt from conda-forge  instead of building them from source.
```suggestion
```